### DISCUSSION TITLE
how to make from source code more easy?

### DISCUSSION THREAD

**Discussion by hxxful** [rizsotto/Bear#393] [Q&A] [answered]
when make, will clone many submodule from github, but my env not allow use the network, can download offline and put packages in somewhere?

**Comment by rizsotto** [comment, marked as answer]
You can install Bear from package and skip the whole build step: https://repology.org/project/bear/versions

You see downloads during build if the dependency is not available, it compiles from source. So, if you can install those packages it won't download anything. https://github.com/rizsotto/Bear/blob/master/INSTALL.md
